### PR TITLE
GA ACS captions

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
GA ACS captions

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: change-beta/@azure-communication-react-7e6e902d-ecc1-4d99-a8e4-6a3abd305ff3.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "prerelease",
```
Comment (JamesBurnside) [inline change-beta/@azure-communication-react-7e6e902d-ecc1-4d99-a8e4-6a3abd305ff3.json:2]: change file should have a stable file and type is `minor` if going from beta -> GA as this now appears in stable builds

File: packages/calling-stateful-client/src/CallSubscriber.ts [hunk 2]
```diff
@@ -120,12 +120,18 @@ export class CallSubscriber {
...
+      this._call.feature(Features.Captions)
+    );
+
     this.subscribe();
   }
 
   private subscribe = (): void => {
     this._call.on('stateChanged', this.stateChanged);
-    this._call.on('stateChanged', this.initCaptionSubscriber);
+    // this._call.on('stateChanged', this.initCaptionSubscriber);
```
Comment (JamesBurnside) [inline packages/calling-stateful-client/src/CallSubscriber.ts:134]: remove line if it's no longer needed

File: packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/ClosedCaptions.test.ts-snapshots/caption-toggle-button-Desktop-Chrome-linux.png
Comment (JamesBurnside) [inline packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/ClosedCaptions.test.ts-snapshots/caption-toggle-button-Desktop-Chrome-linux.png:1]: Why did participant1/2/3 turn into agnes thompson in this snapshot?

Comment (JamesBurnside) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/ClosedCaptions.test.ts-snapshots/caption-toggle-button-Desktop-Chrome-linux.png:1]: And raise hand icon changed also

Comment (carocao-msft) [reply packages/react-composites/tests/browser/snapshots/stable/tests/browser/call/hermetic/ClosedCaptions.test.ts-snapshots/caption-toggle-button-Desktop-Chrome-linux.png:1]: Participant name was a bug fix for ACS captions, the captions name should match with the participant name. Not sure why the raise hand icon changed though...

**Review by JamesBurnside — APPROVED** [review]

**Review by edwardlee-msft — APPROVED** [review]

**Pull request merged**